packages/frontend/src/redux/actions/account.js

```diff
@@ -538,27 +535,15 @@ export const handleCreateAccountWithSeedPhrase = (accountId, recoveryKeyPair, fu
 
 export const finishAccountSetup = () => async (dispatch, getState) => {
     await dispatch(refreshAccount());
-    await dispatch(getBalance());
     await dispatch(clearAccountState());
 
-    const balance = selectBalance(getState());
     const redirectUrl = selectAccountUrlRedirectUrl(getState());
     const accountId = selectAccountId(getState());
 
-    let promptTwoFactor = await TwoFactor.checkCanEnableTwoFactor(balance);
```

Comment:
Removed the conditional prompt of 2FA in the account creation flow in order to remove any bias towards 2FA.